### PR TITLE
Name the parameters and variables in authenticatorGetAssertion.

### PR DESCRIPTION
**Pull request by jyasskin**
***
[Preview](https://s3.amazonaws.com/pr-preview/jyasskin/webauthn/name-get-assertion-params.html#op-get-assertion) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/webauthn/d89c503...jyasskin:5bb3cb0.html)

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1705,46 +1705,56 @@ When this operation is invoked, the authenticator must perform the following pro
...
     [=authenticator=] if it has its own output capability, or by the user agent otherwise.
-1. Process all the supported extensions requested by the client, and generate the [=authenticator data=] as specified in
-    [[#sec-authenticator-data]], though without [=attestation data=]. Concatenate this [=authenticator data=] with the [=hash of
-    the serialized client data=] to generate an [=assertion signature=] using the [=credential private key|private key=] of the
-    selected [=public key credential|credential=] as shown in [Figure 2](#fig-signature), below. A simple, undelimited
+1. Let |processedExtensions| be the result of [=authenticator extension processing=] for each supported extension
+    identifier/input pair in |extensions|.
+1. Let |authenticatorData| be the byte array specified in [[#sec-authenticator-data]] without [=attestation data=] but with any
+    |processedExtensions|.
+1. Let |signature| be the [=assertion signature=] of the concatenation <code>|authenticatorData| || |hash|</code> using the
```
Comment (equalsJeffH) [inline index.bs]: I suppose double-vertical-bar is ok here, but note issue #562.

File: index.bs [hunk 2]
```diff
@@ -1705,46 +1705,56 @@ When this operation is invoked, the authenticator must perform the following pro
...
 
-On successful completion, the authenticator returns to the user agent:
-- The identifier of the credential (credential ID) used to generate the [=assertion signature=].
-- The [=authenticator data=] used to generate the [=assertion signature=].
-- The [=assertion signature=].
-- The user handle associated with the credential used to generate the [=assertion signature=].
+1. If any error occurred while generating the [=assertion signature=], return an error code equivalent to "{{UnknownError}}" and
+    terminate the operation.
+1. Return to the user agent:
+    - |selectedCredential|'s ID
```
Comment (equalsJeffH) [inline index.bs]: i think it is important here to s/ID/credential ID/  -- we are consistently (more or less) using that term and need to more formally define it, so being consistent will help with the latter.

File: index.bs (hunk 1 above)
Comment (jyasskin) [reply index.bs]: Fine: #615 ;)

File: index.bs (hunk 2 above)
Comment (jyasskin) [reply index.bs]: Done.

**Comment by equalsJeffH** [comment]
approved, modulo doing merge-from-master

**Comment by jcjones** [comment]
This looks good to me too, modulo the rebase/merge from master.

**Comment by jcjones** [comment]
Closed by 6594dc7

**Pull request closed without merging**